readme.md

```diff
@@ -126,18 +127,22 @@ function Counter() {
     <h1>
       {count}
       <button onClick={decrement}>Decrease</button>
+      ...
 ```
 
 ### Write only atoms
 
 Just do not define a read function.
 
 ```jsx
-const multiplyCountAtom = atom(null, (get, set, by) => set(countAtom, get(countAtom) * by))
+const multiplyCountAtom = atom(null, (get, set, by) =>
+  set(countAtom, get(countAtom) * by)
+)
```

Comment:
Can we change the name to make it one line? For example, `multiplyCountAtom` 👉 `multiplyAtom`

Reply:
It's fine ，but if `run prettier`，it will be wrapped. 😂😂😂

Reply:
even if you used the shorter name?

Reply:
Yes, I tried，Unless use `multi`

```js
const multi = atom(null, (get, set, by) => set(countAtom, get(countAtom) * by))

function Controls() {
  const [, multi] = useAtom(multi)
  return <button onClick={() => multi(3)}>triple</button>
}
```

Reply:
Oookay, let's leave it then.

Reply:
👌